src/flattenArray.js

```diff
@@ -0,0 +1,8 @@
+// Javascript function to check level of nesting in an array and flatten it
+
```

Comment:
Nice one here 🎉 
but it would have been better if the comment would be describing also what that regex is doing
but overall  the codes look neat and clean